stix2validator/test/attack_pattern_tests.py

```diff
@@ -4,7 +4,7 @@
 from . import ValidatorTest
 from .. import validate_string, validate_instance
 
-VALID_ATTACK_PATTERN = """
+VALID_ATTACK_PATTERN = u"""
 {
```

Comment:
We should consider standardizing on `from __future__ import unicode_literals` instead.